daemon/containerd/image_list.go

```diff
@@ -254,8 +255,9 @@ type imageFilterFunc func(image images.Image) bool
 // setupFilters constructs an imageFilterFunc from the given imageFilters.
 //
 // TODO(thaJeztah): reimplement filters using containerd filters: see https://github.com/moby/moby/issues/43845
```

Comment:
Are there (more) filters we can delegate to containerd? (Otherwise we could remove this comment) - not a blocker though we can always remove in a follow-up

Reply:
Hm, dangling can be done with c8d too.
Not sure about time and how c8d would compare them. Will check.

Reply:
Yeah currently we can't do any more (except dangling). Posted a code that specified available properties in the issue.

Reply:
Crap, dangling is fun too. There's no fuzzy negation, so we'd need something like this: https://github.com/moby/moby/blob/master/.github/workflows/buildkit.yml#L123 😅

Reply:
😬 yeah, let's not do  that for now 😂